src/xtopcom/xstore/src/xaccount_context.cpp

```diff
@@ -1375,5 +1377,23 @@ xaccount_context_t::get_blockchain_height(const std::string & owner) const {
     return height;
 }
 
+common::xtop_logs_t xaccount_context_t::get_logs() {
+    return logs;
+}
+
+void xaccount_context_t::add_log(common::xtop_log_t log) {
+    logs.emplace_back(log);
+    // print log
+    for (auto l : logs) {

```

Comment:
enclose this for loop into macro
`#if defined(DEBUG)
    for (auto const & l : logs) {...};
#endif`